examples/CustomAssertions/CustomAssertions.Tests.ps1

```diff
@@ -0,0 +1,81 @@
+BeforeAll {
```

Comment:
Make sure this file has a `.bicep` and `.md` file with it. Similar to the documentation that was added in this PR #204

Reply:
I don't think it needs a .bicep file since nothing is being deployed. I agree on the .md, will add

Reply:
Actually, I don't know if an `.md` is necessary too since this file will be deprecated in my next PR (to move the tests into the individual modules). I'll update the `.md`'s for each module to include information about which custom assertions work.

Reply:
Will deal with this in next issue #207.